### ISSUE TITLE
reduceRight wrong example?

### ISSUE THREAD

**Issue by alizhdanov** [sdras/array-explorer#78]
Hello,
I'm not sure, but in `find items -> one item`:

`reduce` returns sum of the array, as I'd expect, while `reduceRight` 'flatten' arrays. Shouldn't it be also sum?

**Comment by sdras** [comment]
Yeah @QWp6t you're right, I should make that a little more clear and easy to understand for beginners, since that's who this resource is for.

**Comment by rimildeyjsr** [comment]
@sdras - hello! Can I work on this issue? 🙂
